.github/workflows/trigger-regression-tests.yml

```diff
@@ -0,0 +1,112 @@
+name: Trigger Regression Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Select the branch to run tests on'
+        required: true
+        default: 'main'
```

Comment:
do we need this at all? We can just use `github.ref_name` which will be the branch selected from the list
![image](https://github.com/user-attachments/assets/95779424-8dd7-4cea-b408-ca4b2a54de6e)

Reply:
changed